src/clj/puppetlabs/services/config/puppet_server_config_core.clj

```diff
@@ -92,6 +93,18 @@
                     "conflict with the values to be "
                     "read from puppet: " key-conflicts))))))
 
+(defn check-master-url-prefix
+  "Ensure that Puppet Server's route for the puppet master matches what is in
+  puppet.conf."
+  [webrouter-config puppet-config]
+  (let [puppet-prefix (:master-url-prefix puppet-config)
+        webrouter-prefix (:puppetlabs.services.master.master-service/master-service webrouter-config)]
```

Comment:
This seems like a rather ugly way to be getting this, but it didn't seem like from here we have a good way to get a reference to the master service, and without that I couldn't figure out how to use any of the helpers such as `get-route`. Would love any input on a better way to do this.